hilti/src/rt/tests/util.cc

```diff
@@ -95,6 +95,36 @@ TEST_CASE("enumerate") {
     CHECK_EQ(input, std::vector<char>(input.size(), ' '));
 }
 
+TEST_CASE("escapeBytes") {
+    SUBCASE("escape_quotes") {
+        // This test is value-parameterized over `quote` and `escape_quotes`.
+        std::string quote;
+        bool escape_quotes{};
+        SUBCASE("true") {
+            escape_quotes = true;
+            quote = "\\\"";
+        }
+        SUBCASE("false") {
+            escape_quotes = false;
+            quote = "\"";
+        }
+
+        CHECK_EQ(escapeBytes("", escape_quotes), "");
```

Comment:
Could use `R"(...)"` for better readability in this test.

Reply:
I now use it in a couple places where it helps.